packages/bigtest/API.md

```diff
@@ -0,0 +1,3 @@
+# BigTest
+
+This is cool stuff {@link createInteractor}, yeah!
```

Comment:
This file is just a placeholder, we'll want to expand this to link to the most relevant parts of the documentation.